### ISSUE TITLE
Scribe

### ISSUE THREAD

**Issue by kaniini** [ophion-project/ophion#3]
The scribe feature is implemented with a new channel mode (yet to be determined).  When enabled, the IRCd will send the channel's messages to a designated scribe server, which will persist the logs to a datastore.

Things to decide:

* Should scribe be able to replay logs directly to a client using IRC messages (if so we need something like InspIRCd's `PUSH` to implement this), or should it be restricted to clients which know how to make web services requests?

* Should we provide a web interface to the scribe service, for guests to optionally peruse logs?

**Comment by MRNFN80** [comment]
> Should scribe be able to replay logs directly to a client using IRC messages (if so we need something like InspIRCd's PUSH to implement this),

Yes. I don't for-see many clients being updated/made to include web services requests but thats not to say they won't.

> Should we provide a web interface to the scribe service, for guests to optionally peruse logs?
Not that important and will take time, up to you. I can see this feature being used however.
